### ISSUE TITLE
multi-threded operation in tileserver

### ISSUE THREAD

**Issue by silviucm** [mapzen/tileserver#6]
Hello,

Just playing around with tileserver as a hobby on the home pc, and I noticed the server spawned by the werkzeug libraries chokes heavily on a single cpu core, while trying to hit the postgis database and render them as json tiles, etc

I wonder whether it would generally benefit if you allow the threaded parameter be read from config.yaml

1) Slightly modify the constructor adding threaded parameter explicitly:

```
run_simple(server_config['host'], server_config['port'], tile_server, threaded=server_config.get('threaded', False),
               use_debugger=server_config.get('debug', False),
               use_reloader=server_config.get('reload', False))
```

2) Allow the threaded entry in config.yaml, e.g.

```
server:
  host: localhost
  port: 8099
  debug: true
  reload: true
  threaded: true
```

I personally noticed a much better distribution of CPU load among cores after that.

Regards
silviu

**Comment by rmarianski** [comment]
Sure, that seems reasonable to me. Do you mind submitting a pull request?

**Comment by silviucm** [comment]
will take a stab at it when I get home, cheers

**Comment by nvkelso** [comment]
Closed via PR #7
